### PR TITLE
Make borders optional for form inputs

### PR DESCRIPTION
**Pull request by chris-pearce**
Fed from this Issue: https://github.com/westfieldlabs/scally/issues/47.

### CODE REVIEW

**Comment by kllevin** [comment]
LGTM

File: core/base/_forms.scss [hunk 1]
```diff
@@ -123,8 +123,21 @@ legend {
   select {
     @include to-rem(padding, $form-text-input-and-select-padding);
     color: $colour-form-input-text;
-    @include to-rem(border-radius, $border-radius);
-    @include to-rem(border, 1 solid $colour-form-input-border);
+
+    // Apply a border but make it optional
+    @if $form-text-input-and-select-border {
+
```
Comment (smgcode) [inline core/base/_forms.scss:129]: @chris-pearce I notice you put a new line after/before your <code>@if</code> &  <code>@else</code>, but I don't see the formatting style in https://github.com/westfield/css_guidelines

Comment (chris-pearce) [inline core/base/_forms.scss:129]: @smgcode cause I haven't updated the guidelines yet.

Comment (smgcode) [inline core/base/_forms.scss:129]: :+1:

**Comment by chris-pearce** [comment]
This closes: #47

**Pull request merged**